_data/harnesses/aggregating-average/kstreams.yml

```diff
@@ -169,3 +169,8 @@ prod:
           render:
             skip: true
 
+    - title: Run your app to Confluent Cloud
+      content:
+        - action: skip
+          render:
+            file: tutorials/change-topic-partitions-replicas/ksql/markup/prod/try-ccloud.adoc
```

Comment:
Does KT support a shared "include" location or can we create one easily.  Having this same file shared across tutorials might be confusing for a future reader.  I trained myself to look through these paths to be sure they point to the files in the same tutorial.  I'm worried it could be misinterpreted as a mistake if they point "across tutorials".  

cc @bbejeck @colinhicks

Reply:
@rspurgeon similar concerns.  

Options:

1. As current: All KT refer to the same file from one KT
2. Create a dummy/shared/generic KT: all KT refer to the same file in this shared folder
3. Re-create the same physical file in all individual KTs
4. Re-create the same physical file in all individual KTs but softlink to the same file from one KT

Reply:
I tagged Bill & Colin because they might know more about the capabilities of the tooling.  I would guess the system might support includes somehow, but I'm not familiar with the tech.

Reply:
@rspurgeon unless I'm missing some subtle differentiation in your suggestion, then "support includes somehow" is by definition supported with `_includes/`, see option 2.  All we have to do is agree on the directory structure.

Reply:
I don't know what the requirements are for the tools though, if you just add folders does it cause any issues? Haven't tested

Reply:
@rspurgeon I will test and update the PR if it works.

Reply:
Each render block's file path is resolved from the `_includes` directory, so it is feasible to share common files as @ybyzek suggests in #2 above.

While the convention has been to create unique files for each tutorial, there is no technical mechanism preventing the use of common files.

Reply:
FWIW I pushed an update that implements option 2.  If we like this option, we just need to agree on subdirectory structure, and I can modify as needed.

Reply:
the structure you derived LGTM

Reply:
I also agree with option 2 from the list above.